packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -424,9 +464,10 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
 
     const community = {
       id: payload.id,
-      peerList: [...new Set([localAddress, ...payload.peers])],
-      psk: payload.psk,
-      ownerOrbitDbIdentity: payload.ownerOrbitDbIdentity,
+      peerList: [...new Set([localAddress, ...metadata.peers])],
+      psk: metadata.psk,
+      ownerOrbitDbIdentity: metadata.ownerOrbitDbIdentity,
+      inviteData,
```

Comment:
Sorry for the confusion, I think we should leave this out of the backend model for now and just treat it temporary for now until we are sure we want to store it.